google/auth/exceptions.py

```diff
@@ -34,3 +34,7 @@ class UserAccessTokenError(GoogleAuthError):
 
 class DefaultCredentialsError(GoogleAuthError):
     """Used to indicate that acquiring default credentials failed."""
+
+
+class MutualTlsChannelError(GoogleAuthError):
```

Comment:
Per PEP8, please rename to `MutualTLSChannelError`

https://www.python.org/dev/peps/pep-0008/

> Note: When using acronyms in CapWords, capitalize all the letters of the acronym. Thus HTTPServerError is better than HttpServerError

Reply:
done.